### DISCUSSION TITLE
MELCloud alongside?

### DISCUSSION THREAD

**Discussion by Hapattaja** [SwiCago/HeatPump#213] [Ideas]
Would it be possible to use MELCloud WiFi adapter alongside (connects also to CN105)?
This would make it possible to use both MELCloud and this cloud-free solution.

**Comment by apppie123** [comment]
Not possible with repo.
I saw a product called "ClimaControl" on [Lectronz](https://lectronz.com/products/climacontrol-hvac-mitsubishi-electric) which claims to be able do this, so I suppose it is possible.

**Reply by Hapattaja** [reply]
A great find. Perhaps it has some kind of pass-through mechanism (repeater) implemented programmatically. (A wireless sensor with a battery is just not very reliable, but I think it's optional.)
